### PR TITLE
Add spellchecker_select_languages, reorg scp

### PR DESCRIPTION
**Pull request by tylerkelly13**

### CODE REVIEW

File: plugins/tinymcespellchecker.md [hunk 1]
```diff
@@ -112,85 +117,99 @@ tinymce.init({
 
 {% include configuration/languages.md %}
 
-### `spellchecker_dialog`
+### `spellchecker_languages`
 
-This option specifies the primary spell checking mode. 
+This option specifies the spellchecker languages that are available to the user, provided as a comma delimited string. The specified languages will be available for the spellchecker when using Spell Checker Pro.
```
Comment (metricjs) [inline plugins/tinymcespellchecker.md]: The end of this paragraph reads a bit awkwardly?

File: plugins/tinymcespellchecker.md [hunk 2]
```diff
@@ -112,85 +117,99 @@ tinymce.init({
...
-  spellchecker_dialog: true
+  spellchecker_on_load: true
 });
 ```
 
-### `spellchecker_whitelist`
+### `spellchecker_rpc_url`
 
-This option specifies an array of words to be ignored by the spell checker.
+This option specifies the URL of the server-side `ephox-spelling` service. For instructions on how to setup a Spell Checker Pro server-side component, see: the [server-side component installation guide]({{site.baseurl}}/enterprise/server/).
```
Comment (metricjs) [inline plugins/tinymcespellchecker.md]: "setup" to "set up" in this case maybe? I'm never quite sure with this one

File: plugins/tinymcespellchecker.md (hunk 1 above)
Comment (tiny-james) [reply plugins/tinymcespellchecker.md]: It kinda doesn't need the second sentence.

File: plugins/tinymcespellchecker.md [hunk 3]
```diff
@@ -112,85 +117,99 @@ tinymce.init({
...
 });
 ```
 
-### `spellchecker_on_load`
-
-> **Note**: Removed in Spell Checker Pro 2.0 (TinyMCE 5.2). Spell Checker Pro will now always run on editor initialization. To disable Spell Checker Pro on load, use [`spellchecker_active`](#spellchecker_active).
+### `spellchecker_select_languages`
 
-This option runs the spellchecker when the contents of the editor is loaded.
+This option specifies the languages that can be set for content when working with multi-language content. The specified languages will be available from the [`language` toolbar drop-down menu button](#toolbarbuttons).
```
Comment (tiny-james) [inline plugins/tinymcespellchecker.md]: You might want to mention that unlike the other list of languages this one can take any language, not just the ones that the spellchecker actually supports.

File: plugins/tinymcespellchecker.md [hunk 4]
```diff
@@ -112,85 +117,99 @@ tinymce.init({
...
-This option specifies the primary spell checking mode. 
+This option specifies the spellchecker languages that are available to the user, provided as a comma delimited string. The specified languages will be available for the spellchecker when using Spell Checker Pro.
 
-- When set to `true`, the [`spellcheck` toolbar button](#toolbarbuttons) will open a dialog that will step the user through each potential spelling error in the document. 
-- When set to `false`, the [`spellcheck` toolbar button](#toolbarbuttons) will enable or disable as-you-type spell checking and the [`spellcheckdialog` toolbar button](#toolbarbuttons) will open the spell checker dialog.
+**Type:** comma-separated `String`
+
+**Default Value:**
+````
+'US English=en_us,UK English=en_gb,Danish=da,Dutch=nl,Finnish=fi,French=fr,German=de,Italian=it,Norwegian=nb,Brazilian Portuguese=pt,Iberian Portuguese=pt_PT,Spanish=es,Swedish=sv'
```
Comment (tiny-james) [inline plugins/tinymcespellchecker.md]: You might want to mention the list of languages supported by the spellchecker service as it's pretty limited.

File: plugins/tinymcespellchecker.md (hunk 2 above)
Comment (tylerkelly13) [reply plugins/tinymcespellchecker.md]: correct. "set up" fixed

File: plugins/tinymcespellchecker.md (hunk 4 above)
Comment (tylerkelly13) [reply plugins/tinymcespellchecker.md]: Added:
> For a list of available languages, see: [Supported languages](#supportedlanguages).

File: plugins/tinymcespellchecker.md (hunk 3 above)
Comment (tylerkelly13) [reply plugins/tinymcespellchecker.md]: checked with @jhaines, users should only specify languages supported by spell checker pro, so I've added a link to the supported languages table on the same page.

File: plugins/tinymcespellchecker.md (hunk 1 above)
Comment (tylerkelly13) [reply plugins/tinymcespellchecker.md]: removed second sentence.

**Pull request merged**